.github/workflows/build.yml

```diff
@@ -11,7 +11,7 @@ on:
     inputs:
       toolchain:
         description: 'Default Rust Toolchain'
-        default: "1.78.0"
+        default: "1.75.0"
```

Comment:
i'm curious why downgrade it?

Reply:
The thing is, cosmwasm-std package < 2.0.2 is incompatible with rust 1.78+ https://github.com/CosmWasm/cosmwasm/issues/2061. We can easily update cosmwasm-std to latest version, we just need to reword contract dependencies.